app/routes/archive/components/ItemArchivePanel.jsx

```diff
@@ -17,7 +17,7 @@ const ItemArchivePanel = ({
 }) => {
   let progressElement
 
-  if (cloneProgress === 0) {
+  if (cloneProgress === 0 && !cloneStatus) {
```

Comment:
What is `cloneStatus`?

Reply:
`cloneStatus` is any message we dispatch to the store to show for the submission being cloned

Example: If a clone fails we dispatch the `cloneStatus` of a submission to be "Clone failed, an error has occurred"

There was a bug here where we were showing the spinner even after the folder had errored when cloning (noticed on Windows)